nodebuilder/tests/swamp/config.go

```diff
@@ -35,3 +37,20 @@ func WithBlockTime(t time.Duration) Option {
 		c.Tendermint.Consensus.SkipTimeoutCommit = false
 	}
 }
+
+// WithLogLevel overrides basic log levels for test purposes
+func WithLogLevel(components []string, levels []string) Option {
+	return func(_ *Components) {
+		if len(components) != len(levels) && len(levels) != 1 {
```

Comment:
```suggestion
		if len(levels) == 0 || len(components) == 0 || len(levels) != len(components) {
```

Reply:
instead of passing the log level for each component, you can pass only one that will be applied for all.

Reply:
So this condition means that you can have log level for each component or one for all components. Otherwise, panic